spec/dependent_styles_spec.rb

```diff
@@ -1,82 +1,79 @@
-Dependents.each_pair do |id, (filename, path, style, reason)|
+Dependents.each_pair do |basename, (filename, path, style, reason)|
 
-  describe "dependent style #{id}" do
+  describe "#{basename} (dependent):" do
```

Comment:
Maybe it helps to clarify here
```#{basename} (dependent subdirectory)```
This will make the mentions of subdirectory below unneccessary.